src/mero_wrk_tcp_binary.erl

```diff
@@ -176,6 +192,9 @@ pack({?MEMCACHE_INCREMENT, {Key, Value, Initial, ExpTime}}) ->
 pack({?MEMCACHE_DELETE, {Key}}) ->
     pack(<<>>, ?MEMCACHE_DELETE, Key);
 
+pack({?MEMCACHE_DELETEQ, {Key}}) ->
```

Comment:
Quite right about the use of pmap. An oversight.
On Mon, Feb 15, 2016 at 21:47 Brian Troutwine <brian@troutwine.us> wrote:

> Quite right. I'd neglected to remove the DELETEQ references or investigate
> their use but left it in a half-state when I settled on using DELETE.
> On Mon, Feb 15, 2016 at 19:00 Miriam Pena <notifications@github.com>
> wrote:
>
>> In src/mero_wrk_tcp_binary.erl
>> <https://github.com/AdRoll/mero/pull/13#discussion_r52958153>:
>>
>> > @@ -176,6 +192,9 @@ pack({?MEMCACHE_INCREMENT, {Key, Value, Initial, ExpTime}}) ->
>> >  pack({?MEMCACHE_DELETE, {Key}}) ->
>> >      pack(<<>>, ?MEMCACHE_DELETE, Key);
>> >
>> > +pack({?MEMCACHE_DELETEQ, {Key}}) ->
>>
>> this one is not used.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/AdRoll/mero/pull/13/files#r52958153>.
>>
>